module-services/service-db/include/service-db/QuotesMessages.hpp

```diff
@@ -55,6 +55,37 @@ namespace Quotes
         }
     };
 
+    struct IdQuoteAuthorRecord
+    {
+        unsigned int quote_id;
+        std::string quote;
+        std::string author;
+
+        IdQuoteAuthorRecord() = default;
+
+        explicit IdQuoteAuthorRecord(QueryResult *query)
+        {
+            quote_id = (*query)[0].getInt32();
```

Comment:
Is there any possibility that `getInt32()` returns negative value? If there is, wraparound will occur as you store its return value in unsigned type

Reply:
Not really, because this object is created after a response from DB which succeeded, but you are right, the quote_id should be `std::int32_t`. Thanks ;)

Reply:
Done